pinot-tools/src/main/java/org/apache/pinot/tools/data/generator/DataGenerator.java

```diff
@@ -110,7 +110,7 @@ public void generateCsv(long totalDocs, int numFiles)
       throws IOException {
     final int numPerFiles = (int) (totalDocs / numFiles);
     for (int i = 0; i < numFiles; i++) {
-      try (FileWriter writer = new FileWriter(outDir + "/output.csv")) {
+      try (FileWriter writer = new FileWriter(String.format("%s/output_%d.csv", outDir, i))) {
```

Comment:
Do not hard-code the file separator
```suggestion
      try (FileWriter writer = new FileWriter(new File(outDir, String.format("output_%d.csv", i)))) {
```